src/pages/Animeworld/main.ts

```diff
@@ -0,0 +1,46 @@
+import { pageInterface } from '../pageInterface';
+
+export const Animeworld: pageInterface = {
+  name: 'Animeworld',
+  domain: 'https://www.animeworld.tv',
+  languages: ['Italian'],
+  type: 'anime',
+  isSyncPage(url) {
+    if (url.split('/')[4] !== undefined && url.split('/')[4].length) {
+      return true;
+    }
+    return false;
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('#anime-title').text().trim();
+    },
+    getIdentifier(url) {
+      return utils.urlPart(url, 4);
+    },
+    getOverviewUrl(url) {
+      return `${Animeworld.domain}/${Animeworld.sync.getIdentifier(url)}`;
+    },
+    getMalUrl(provider) {
+      return new Promise(resolve => {
+        resolve(j.$('#mal-button').attr('href') || false);
```

Comment:
i think you misunderstood what im trying to say
https://github.com/MALSync/MALSync/blob/master/src/pages/animepahe/main.ts#L52
i mean it like this
so mal first anilist if you use anilist second

Reply:
ahh got it, sorry i misunderstood